docs/docs/advanced/debugging.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 10
+title: 🐛 Debugging
+---
+
+In some cases, you'll want to debug your Dart Frog app, and you might have noticed that by adding just the breakpoint, the debugger doesn't stop there like it does by default in a Flutter app.
+Below, there are two quick and easy options for debugging code in Dart Frog.
+
+## Debugging with Dart Frog IDE Extension 🐸
+
+:::caution
+This extension is only available for Visual Studio Code, if you are using another IDE, please refer to the **Debugging by Attaching to Dart Process** section.
+:::
+
+:::info
+If you are interested in an extension for Android Studio or OpenVSX there are some open issues you can follow to learn more about it.
+
+- **Android Studio** [#1326](https://github.com/VeryGoodOpenSource/dart_frog/issues/1326)
+
+- **OpenVSX** [#907](https://github.com/VeryGoodOpenSource/dart_frog/issues/907)
+
+:::
+
+- Go to the Visual Studio Code Marketplace in [here](https://marketplace.visualstudio.com/items?itemName=VeryGoodVentures.dart-frog) and install the extension.
```

Comment:
The Dart Frog extension requires the Dart extension to be installed previously in the IDE. Maybe we can add a step detailing such. 

In most cases if the user doesn't have the Dart extension installed this should happen automatically since it is listed as a required dependency: https://github.com/VeryGoodOpenSource/dart_frog/blob/a9e2a1c7c1742e22829b3d7c580d5188c807c9b4/extensions/vscode/package.json#L148